specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -153,7 +153,22 @@ Consider at least the below topics but you do not need to cover those that are n
       * Is it logged?
   * What secrets are needed by the components? How are these secrets secured and attained?
 -->
-
+### CICD
+
+#### e2e test image
+To run e2e tests alot of dependencies are needed (browsers, build systems etc...).
```

Comment:
what is etc? what dependencies are needed, and for what purpose?

Reply:
There is a more detailed list below? I added details beside list below. 
Let me know if that looks good?

Reply:
looks ambiguously stated, since a person reads from start-to-end usually. I want to make sure the VEP answers questions rather tnan opens more. In case the comment is no helpful/spawning any changes, you can resolve it 😊

Reply:
I think that its states the problem clearly before stating the solution. 
It gives a wider context before digging into the details.